### ISSUE TITLE
Android - crash because loginContext is null in onCompleted

### ISSUE THREAD

**Issue by NGumby** [jeduan/cordova-plugin-facebook4#144]
I got this Crash report on an Android 5.1 device :

```
java.lang.NullPointerException: Attempt to invoke virtual method 'void org.apache.cordova.CallbackContext.success(org.json.JSONObject)' on a null object reference
	at org.apache.cordova.facebook.ConnectPlugin$2$1.onCompleted(ConnectPlugin.java:114)
	at com.facebook.GraphRequest$1.onCompleted(GraphRequest.java:304)
	at com.facebook.GraphRequest$5.run(GraphRequest.java:1379)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:145)
	at android.app.ActivityThread.main(ActivityThread.java:6897)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199)
```

it looks like this line is causing the problem 
`    loginContext.success(getResponse());
`

So there is a case when loginContext is set to null, then the onCompleted is called again.  But I can't reproduce it.

Maybe someone has a clue on how this can happen and be fixed.

I'm using plugin version 1.4.1, with cordova 5.3.3 and Android 4.1.1

**Comment by Emadello** [comment]
I have the same issue .. Trying to fix...

**Comment by nootherday** [comment]
I got same issue on plugin version 1.6.3, android 4.4.2

Fatal Exception: java.lang.NullPointerException
       at org.apache.cordova.facebook.ConnectPlugin$2$1.onCompleted(ConnectPlugin.java:116)
       at com.facebook.GraphRequest$1.onCompleted(GraphRequest.java:304)
       at com.facebook.GraphRequest$5.run(GraphRequest.java:1379)
       at android.os.Handler.handleCallback(Handler.java:733)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:136)
       at android.app.ActivityThread.main(ActivityThread.java:5120)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:792)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:608)
       at dalvik.system.NativeStart.main(NativeStart.java)

**Comment by robertarnesson** [comment]
should be fixed @0a5ac31
